docs/docs-content/clusters/edge/hardware-requirements.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_label: "Hardware Requirements"
+title: "Hardware Requirements"
+description: "Learn about the hardware requirements for Palette Edge hosts."
+hide_table_of_contents: false
+sidebar_position: 0
+tags: ["edge", "architecture"]
+---
+
+An Edge host must meet the minimum hardware requirements to be deployed successfully. Certain features such as Trusted
+Boot also specific hardware requirements. This page provides general guidance on the minimum hardware requirements for
+your Edge hosts.
```

Comment:
```suggestion
An Edge host must meet the minimum hardware requirements to successfully deploy Palette. Certain features, such as Trusted
Boot, also have specific hardware requirements. This page provides general guidance on the minimum hardware requirements for
your Edge hosts.
```

Reply:
I think that's the correct meaning? In the first sentence, I wanted to emphasize that it's not about deploying the Edge host itself, but about deploying Palette onto the Edge host (disregard if I'm wrong).

Reply:
It's technical deploying a cluster using Palette. Deploy Palette makes it sound like a self-hosted instance.